pytest_bdd/feature.py

```diff
@@ -53,6 +53,7 @@
     ("Then ", types.THEN),
     ("@", types.TAG),
     ("And ", None),  # Unknown step type,
```

Comment:
and why is it unknown here btw

Reply:
Changing it into # Continuation of the previously mentioned step type